upkie/cpp/actuation/BulletInterface.cpp

```diff
@@ -344,7 +346,27 @@ void BulletInterface::read_contacts() {
         contact_info.m_numContactPoints;
   }
 }
-
+int BulletInterface::environment_collision() {
+  b3ContactInformation contact_info;
+  b3RobotSimulatorGetContactPointsArgs contact_args;
+  const int nb_links = bullet_.getNumJoints(robot_);
+  for (const auto& key_child : body_names) {
```

Comment:
`body_names` does not include the ground plane.

On a side note, we should probably rename `body_names`:

- First, it is a name -> id dictionary, and dictionaries are usually named after their values rather than their keys :see_no_evil: In that case the name would be `body_ids`.
- Second: I was confused at first when reading it (are those robot body names?). We can make it more obvious with a name like `env_bodies` or `env_body_ids`.